RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -63,6 +65,28 @@ PixelThresholdClusterizer::PixelThresholdClusterizer
 /////////////////////////////////////////////////////////////////////////////
 PixelThresholdClusterizer::~PixelThresholdClusterizer() {}
 
+
+// Configuration descriptions
+void
+PixelThresholdClusterizer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  // siPixelClusters
+  edm::ParameterSetDescription desc;
+  desc.add<edm::InputTag>("src", edm::InputTag("siPixelDigis"));
+  desc.add<int>("ChannelThreshold", 1000);
+  desc.addUntracked<bool>("MissCalibrate", true);
+  desc.add<bool>("SplitClusters", false);
+  desc.add<int>("VCaltoElectronGain", 65);
+  desc.add<int>("VCaltoElectronGain_L1", 65);
+  desc.add<int>("VCaltoElectronOffset", -414);
+  desc.add<int>("VCaltoElectronOffset_L1", -414);
+  desc.add<std::string>("payloadType", "Offline");
+  desc.add<int>("SeedThreshold", 1000);
+  desc.add<int>("ClusterThreshold_L1", 4000);
+  desc.add<int>("ClusterThreshold", 4000);
+  desc.add<int>("maxNumberOfClusters", -1);
+  descriptions.add("siPixelClusters", desc);
+}
```

Comment:
If I'm not mistaken, currently this method is a dead code.
Perhaps in the future refactoring this can become useful.